jetpack-helper.php

```diff
@@ -63,7 +63,7 @@ function jetpack_amp_build_stats_pixel_url() {
 		$data = compact( 'v', 'j', 'blog', 'post', 'tz', 'srv' );
 	}
 
-	$data['host'] = isset( $_SERVER['HTTP_HOST'] ) ? rawurlencode( $_SERVER['HTTP_HOST'] ) : ''; // input var ok
+	$data['host'] = isset( $_SERVER['HTTP_HOST'] ) ? sanitize_text_field( wp_unslash( $_SERVER['HTTP_HOST'] ) ) : ''; // input var ok
```

Comment:
`sanitize_text_field` is not equivalent to `rawurlencode`

Reply:
Nevermind. I supose this change is ok. You can cherry-pick this into #1002 as well.